apps/styleguide/src/forms/InputOntology.vue

```diff
@@ -63,6 +63,9 @@
           />
         </span>
       </div>
+      <span v-if="Object.keys(terms).length > 50 && searchResultCount >= 0">
```

Comment:
This was created to help testing, but actually it is kind of cool so I kept it. What do you think?